README.md

```diff
@@ -62,7 +62,7 @@ as well as [commercial GeoIP2 databases](https://www.maxmind.com/en/geoip2-datab
 
 After installing this module and obtaining a database, you must now set up the
 module in your Apache configuration file (e.g., `/etc/apache2/apache2.conf`)
-or in an `.htaccess` file. You must set `MaxMindDBEnable` to enable the
+or in an `.htaccess` file. You must load the module with `LoadModule maxminddb_module /usr/lib/apache2/modules/mod_maxminddb.so`. You must set `MaxMindDBEnable` to enable the
```

Comment:
How about having this in a separate section, for example before `## Usage`? Such as `## Loading the Module`. I suggest this because I think there is more to say about it (the paths may be different for different installs, and we could probably mention `a2enmod` for Debian based users). This section as written is more about working with it once it's loaded which I think could continue to be discussed separately.

Reply:
Sure, you can put this anywhere